src/lua/crypto_ethereum.lua

```diff
@@ -285,6 +284,9 @@ function ETH.make_storage_data(src)
    offset = O.from_hex('0000000000000000000000000000000000000000000000000000000000000020')
 
    -- length as a 256 unsigned integer
+   if iszen(type(src)) then
```

Comment:
I am not 100% sure of the way arguments are used in lua, as references or not. However it is safer here to make src as a const and instead of overwriting its contents to create a new local variable like src_o